packages/unlock-js/src/walletService.ts

```diff
@@ -117,14 +117,14 @@ export default class WalletService extends UnlockService {
    * Function which yields the address of the account on the provider
    */
   async getAccount() {
-    const account = await this.provider.getSigner()
+    const account = await this.provider.getSigner(0)
 
-    if (!account.length) {
+    if (!account.address) {
```

Comment:
For ethers 6, the getter for the address as become an asynchronous method so you may need to call `await account.getAddress()ˋ instead